sig-ui/charter.md

```diff
@@ -0,0 +1,44 @@
+# SIG UI Charter
+
+This charter is a WIP.
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and uses the Roles and Organization Management outlined in [sig-governance].
+
+## Scope
+
+SIG-UI covers all things user interface related within the Kubernetes project. Efforts are centered around the Kubernetes Dashboard: a general purpose, web-based UI for Kubernetes clusters. It allows users to manage applications running in the cluster and troubleshoot them, as well as manage the cluster itself.
+
+### In scope
+
+  Link to SIG section in [sigs.yaml]
+
+#### Code, Binaries and Services
+
+- Kubernetes Dashboard
+
+#### Cross-cutting and Externally Facing Processes
+
+- Cutting a new release of the Dashboard
+
+### Out of Scope
+
+Tools that contributors use in support of the project (eg. Prow, Test Grid)
```

Comment:
When you say "all things user interface related" you probably don't mean the kubernetes.io website, the blog?

How do you interface with the CNCF when it comes to branding around Kubernetes concepts, ie: if there were a standard icon for Pods who would develop and own that?

When we talk about "UI" we're probably implicitly talking about "GUI", and user interfaces on the command line fall under sig-cli?

Reply:
That's a good point @spiffxp. Clarifying as *GUI* rather than UI would be good. 

Branding hasn't been SIG-UI and I think makes sense to not be included. As far as diagrams and iconography, that's been explored by SIG-UI in the past (e.g. #747), but under the umbrella of GUI.

Reply:
+1 to s/UI/GUI